src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -300,6 +301,10 @@ export default {
 			type: String,
 			default: '/basket'
 		},
+		customCheckoutButtonAction: {
```

Comment:
Doesn't seem like this is used in this component. Do you plan to call it somewhere in this file later?

Reply:
No, you're right this doesn't belong here anymore.  I just deleted it.